NORDIC Embedded Software/nrf52 Patient Monitoring Device/main.c

```diff
@@ -617,20 +614,20 @@ int main(void)
     advertising_init();
     conn_params_init();
 	
-		printf("Basic ini done\r\n"); 
-		nrf_delay_ms(500);
+    printf("Basic ini done\r\n"); 
```

Comment:
Consider to use nrf_log system instead of printf to avoid situation when log systems fight for access to UART/RTT system